site/cypress/e2e/Cypress-Gradeable/rainbow_grading.spec.js

```diff
@@ -4,18 +4,164 @@ skipOn(Cypress.env('run_area') === 'CI', () => {
+            checkTextbox('[data-testid="section-and-labels-10"]', '10', 'TA 10');
+            checkTextbox('[data-testid="cutoff-A"]', '93', '87');
+            checkTextbox('[data-testid="cutoff-A-"]', '90', '80');
+            checkTextbox('[data-testid="cutoff-B+"]', '87', '77');
+            checkTextbox('[data-testid="cutoff-B"]', '83', '73');
+            checkTextbox('[data-testid="cutoff-B-"]', '80', '70');
+            checkTextbox('[data-testid="cutoff-C+"]', '77', '67');
+            checkTextbox('[data-testid="cutoff-C"]', '73', '63');
+            checkTextbox('[data-testid="cutoff-C-"]', '70', '60');
+            checkTextbox('[data-testid="cutoff-D+"]', '67', '55');
+            checkTextbox('[data-testid="cutoff-D"]', '60', '50');
+
+            // Ensure tables can be added to and removed from
+            // TODO: add checks for other tables once features are implemented
+            cy.get('[data-testid="manual-grading-user-id"]').type('adamsg');
+            cy.get('[data-testid="manual-grading-grade"]').select(1);
+            cy.get('[data-testid="manual-grading-note"]').type('MESSAGE');
+            cy.get('[data-testid="manual-grading-submit"]').click();
+            cy.get('[data-testid="manual-grading-table-body"] > tr > td').as('manual-grading-table-elements');
+            cy.get('@manual-grading-table-elements').eq(0).should('contain', 'adamsg');
+            cy.get('@manual-grading-table-elements').eq(1).should('contain', 'A');
+            cy.get('@manual-grading-table-elements').eq(2).should('contain', 'MESSAGE');
+            cy.get('@manual-grading-table-elements').eq(3).find('a').click();
+
+            cy.get('[data-testid="plagiarism"]').should('be.visible'); // Visibility not based on checkbox
+            cy.get('[data-testid="plagiarism-user-id"]').type('adamsg');
+            cy.get('[data-testid="plagiarism-gradeable-id"]').select(1);
+            cy.get('[data-testid="plagiarism-marks"]').type('1');
+            cy.get('[data-testid="plagiarism-submit"]').click();
+            cy.get('[data-testid="plagiarism-table-body"] > tr > td').as('plagiarism-table-elements');
+            cy.get('@plagiarism-table-elements').eq(0).should('contain', 'adamsg');
+            cy.get('@plagiarism-table-elements').eq(1).should('contain', 'grades_released_homework_autota');
+            cy.get('@plagiarism-table-elements').eq(2).should('contain', '1');
+            cy.get('@plagiarism-table-elements').eq(3).find('a').click();
+        });
+        it('Manual Customization upload should work', () => {
+            // Upload manual customization
+            cy.get('[data-testid="btn-upload-customization"]').should('exist');
+            cy.get('[data-testid="config-upload"]').should('exist');
+            // TODO: select file using the Upload button instead of force clicking a hidden element
+            cy.get('[data-testid="config-upload"]').selectFile('cypress/fixtures/manual_customization.json', { force: true });
```

Comment:
If you aren't planning on changing the force click to upload without force, can you create an issue to do this one specifically?